pxtrunner/runner.ts

```diff
@@ -596,9 +596,11 @@ ${files["main.ts"]}
                         // Split the steps
                         let stepcontent = content.innerHTML.split(/<h3.*\/h3>/gi);
                         for (let i = 0; i < stepcontent.length - 1; i++) {
-                            stepInfo[i].headerContent = stepcontent[i + 1].split(/(<.*?>.*<\/.*?>)/i)[1];
+                            content.innerHTML = stepcontent[i + 1];
```

Comment:
There is most definitely a library, but I don't see a need to add another dependency. 
I have figured out a way to use the content element (ie: the passed in DOM) to do the parsing.